.jenkins/pytorch/build.sh

```diff
@@ -100,5 +100,5 @@ if [[ "$BUILD_TEST_LIBTORCH" == "1" ]]; then
   echo "Building libtorch"
   # NB: Install outside of source directory (at the same level as the root
   # pytorch folder) so that it doesn't get cleaned away prior to docker push.
-  WERROR=1 VERBOSE=1 tools/cpp_build/build_all.sh "$PWD/../cpp-build"
+  TORCH_WERROR=1 VERBOSE=1 tools/cpp_build/build_all.sh "$PWD/../cpp-build"
```

Comment:
What's going on here?

Reply:
now that torch is built via add_subdirectory from caffe2, it's exposed to a new set of `-W...` flags that breaks the build in CI. Given that we already have a tech debt item to clean up and homogenize Werror usage once all the dust settles, having torch not share the exact same WERROR flag with caffe2 is an easy way to unblock this diff.

Reply:
I think it would be easier/better to use the same set of warning flags for the whole project, and then add the warnings back. Since this build has always been separate, I could do whatever I wanted, including adding all the warnings I wanted. If we're unifying, we should use the same warnings with the same `WERROR` everywhere. So you should instead just remove the additional warnings that break the CI